arraycontext/impl/pytato/fake_numpy.py

```diff
@@ -23,9 +23,10 @@
 """
 from functools import partial, reduce
 
-from arraycontext.fake_numpy import (
-        BaseFakeNumpyNamespace, BaseFakeNumpyLinalgNamespace,
-        )
+from arraycontext.fake_numpy import \
+        BaseFakeNumpyLinalgNamespace
+from arraycontext.loopy import \
+        LoopyBasedFakeNumpyNamespace
```

Comment:
Please avoid unnecessary diffs while submitting patches.

Reply:
I was getting the "unused import" error because of this but I can revert back to the original one.